modules/031-local-path-provisioner/images/local-path-provisioner/werf.inc.yaml

```diff
@@ -1,34 +1,42 @@
 {{- $localPathProvisionerVersion := "0.0.26"}}
-
 ---
 image: {{ $.ModuleName }}/{{ $.ImageName }}
 fromImage: common/distroless
 import:
-  - artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
-    add: /local-path-provisioner
-    to: /usr/bin/local-path-provisioner
-    after: install
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
+  add: /local-path-provisioner
+  to: /usr/bin/local-path-provisioner
+  after: install
 docker:
   ENTRYPOINT: ["/usr/bin/local-path-provisioner"]
 ---
-artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
-from: {{ $.Images.BASE_GOLANG_21_ALPINE_DEV }}
+artifact: {{ .ModuleName }}/{{ .ImageName }}-src-artifact
+fromArtifact: src-artifact
 git:
-  - add: /{{ $.ModulePath }}modules/031-{{ $.ModuleName }}/images/{{ $.ImageName }}/patches
-    to: /patches
-    stageDependencies:
-      install:
-        - '**/*'
+- add: /{{ $.ModulePath }}/modules/031-{{ $.ModuleName }}/images/{{ $.ImageName }}/patches
+  to: /patches
+  stageDependencies:
+    install:
+      - '**/*'
+shell:
+  install:
+  - git clone --depth 1 --branch v{{$localPathProvisionerVersion}} {{ $.SOURCE_REPO }}/rancher/local-path-provisioner.git /src
+  - cd /src
+  - git apply /patches/*.patch --verbose 
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
+from: {{ $.Images.BASE_GOLANG_23_ALPINE }}
+import:
+- artifact: {{ .ModuleName }}/{{ $.ImageName }}-src-artifact
+  add: /src
+  to: /src
+  before: install
 mount:
-  - fromPath: ~/go-pkg-cache
-    to: /go/pkg
+- fromPath: ~/go-pkg-cache
+  to: /go/pkg
 shell:
   install:
-    - mkdir /src
-    - git clone --depth 1 --branch v{{$localPathProvisionerVersion}} {{ $.SOURCE_REPO }}/rancher/local-path-provisioner.git /src
     - cd /src
-    - for ifile in $(ls /patches/*.patch); do git apply $ifile; done
-    - go get github.com/prometheus/client_golang@v1.17.0
     - go mod tidy
     - go mod vendor
     - GOPROXY={{ $.GOPROXY }} CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "-X main.VERSION={{$localPathProvisionerVersion}} -extldflags -static -s -w" -o /local-path-provisioner
```

Comment:
GOPROXY needs to be before go mod tidy /go mod vendor

Reply:
done